### PR TITLE
Autocomplete

### LINKED ISSUE #19: Move the components from Innopoints
The following components are to be moved from Innopoints:
- [x] Accordion and AccordionSection
- [x] Autocomplete
- [x] Button
- [x] Card (add an outlined version too)
- [x] CheckboxChipGroup
- [x] CheckboxGroup
- [x] DatePicker
- [x] Dialog
- [x] Dot
- [x] Dropdown
- [x] FileInput
- [x] FormField
- [x] Modal
- [x] Popover
- [x] RadioChipGroup
- [x] RadioGroup
- [x] Snackbar
- [x] StarRating
- [x] Switch
- [x] Tabs
- [x] TextField
- [x] TimePicker
- [x] Chip

### PR DESCRIPTION
**Pull request by illright**
Closes #19 
Was pretty reckless with this one, check for any console.logs or stuff like this

### CODE REVIEW

File: src/_utils/call-on-sight.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* Action that allows calling a function when an element appears in the viewport.
+   Uses the IntersectionObserver API, which might not be available in older browsers like IE11,
+   If it is not available, the action will be silently ignored. */
+export default function callOnSight(node, { callback, args = [] }) {
```
[the remark this answers is not in the document]
Comment (illright) [reply src/_utils/call-on-sight.js:4]: There isn't really a way to implement the same behaviour as in `setTimeout` due to the way the action parameters are passed

[the remark this answers is not in the document]
Comment (aabounegm) [reply src/_utils/call-on-sight.js:4]: yeah, I forgot that it passes only one argument. However, you can let it be just the callback and make it the user's responsibility to wrap it in an arrow function or something, as is the case with event listeners, for example

[the remark this answers is not in the document]
Comment (illright) [reply src/_utils/call-on-sight.js:4]: Kinda lame, and for what purpose?

[the remark this answers is not in the document]
Comment (aabounegm) [reply src/_utils/call-on-sight.js:4]: just didn't like how it looks like `use:callOnSight={{ callback }}`

**Pull request merged**